ecs/pod_test.go

```diff
@@ -54,9 +63,61 @@ func TestECSPod(t *testing.T) {
 
 			c, err := NewBasicECSClient(*awsOpts)
 			require.NoError(t, err)
+			defer c.Close(ctx)
 
 			tCase(tctx, t, c)
 		})
+
+	}
+
+}
+
+func TestECSPod(t *testing.T) {
```

Comment:
I don't think this is worth doing now, but can you make a ticket and attach a TODO comment to clean up the resources in this test more thoroughly? Since this test has the potential to leak resources (secrets, task definitions, and tasks), we need to clean up to make sure that after all the tests have finished running, there's no lingering resources in AWS. I would make a comment like this:
`// TODO (<TICKET_NUMBER>): clean up resources in ECS tests more thoroughly.

Reply:
Sure, done.